sys-kernel/gentoo-kernel/gentoo-kernel-5.10.42.ebuild

```diff
@@ -79,12 +81,21 @@ src_prepare() {
 	esac
 
 	echo 'CONFIG_LOCALVERSION="-gentoo-dist"' > "${T}"/version.config || die
+	local dist_conf_path="${WORKDIR}/gentoo-kernel-config-${GENTOO_CONFIG_VER}"
```

Comment:
Consistency nit:
```suggestion
	local dist_conf_path="${WORKDIR}"/gentoo-kernel-config-${GENTOO_CONFIG_VER}
```

Reply:
that's holywar, but I'd leave it fully commented, just couple lines above in src_`prepare` it is.